CONTRIBUTING.md

```diff
@@ -1,41 +1,50 @@
-## Submitting Desktop Client issues
-
-If you have questions about how to use the Nextcloud Desktop Client, please
-visit our [support site][support] or our [forum][forum].
-We are also available on [IRC][irc].
+## Contributing by reporting issues
+* **Always report bugs via Github**: that is the most reliable way to contacts us since we do not follow any process to check the forum and social media. 
+* **For questions about the Nextcloud Desktop Client, use the [📋 Forum](https://help.nextcloud.com)**.
```

Comment:
what do we want to use https://github.com/nextcloud/desktop/discussions for?

Reply:
... there's also the public Desktop Talk channel: https://cloud.nextcloud.com/call/d8k2r6p6

would be a replacement for the IRC for sure